### PR TITLE
Aruha 718 get exhibitor list

### PR DESCRIPTION
**Pull request by lmontrieux**
[ARUHA 718](https://techjira.zalando.net/browse/ARUHA-718)

And if this instance dies - bubuku will die as well, trying to restart kafka without any need for that

### CODE REVIEW

**Comment by adyach** [comment]
@lmontrieux I see that it handles exception while connecting to zookeeper, which should help, but it is not clear how it helps to connect to different zookeepers.

**Comment by lmontrieux** [comment]
@adyach it's in a loop that goes through all zookeepers. The issue we had was that the connection attempt would throw a ConnectionError, which wasn't caught, and so we would never try to connect to the other zookeepers.

**Comment by adyach** [comment]
👍

**Comment by adyach** [comment]
we discussed with @lmontrieux offline, everything looks fine, except this deadline for 120 secs. we decided to test it as it is.

**Comment by lmontrieux** [comment]
👍

File: bubuku/zookeeper/exhibitor.py [hunk 1]
```diff
@@ -35,4 +35,8 @@ def _query_exhibitors(self, exhibitors):
                 return response.json()
             except RequestException as e:
                 _LOG.warn('Failed to query zookeeper list information from {}'.format(url), exc_info=e)
+            except ConnectionError as e:
+                _LOG.warn('Failed to connect to zookeeper instance {}'.format(url), exc_info=e)
+            except:
+                _LOG.warn('Unknown error connecting to zookeeper instance {}'.format(url), exc_info=e)
```
Comment (CyberDem0n) [inline bubuku/zookeeper/exhibitor.py]: e is not defined here...

Comment (lmontrieux) [inline bubuku/zookeeper/exhibitor.py]: Thanks :) https://github.com/zalando-incubator/bubuku/pull/82/commits/fd46d076431061790fff16f293662a489e577ddf

**Comment by v-stepanov** [comment]
👍

**Comment by lmontrieux** [comment]
👍

**Pull request merged**